spec/helpers.lua

```diff
@@ -122,8 +122,25 @@ do
     end
 end
 
-local function get_db_utils(strategy, no_truncate)
+local function truncate_tables(tables)
+  if not tables then
+    return
+  end
+
+  for _, t in ipairs(tables) do
+    if dao[t] then
+      dao:truncate_table(t)
+    else
+      db:truncate(t)
+    end
+  end
+end
+
+local function get_db_utils(strategy, tables)
   strategy = strategy or conf.database
+  if tables ~= nil and type(tables) ~= "table" then
+    error("arg #2 must be a list of tables to truncate")
```

Comment:
very minor because a spec utils, but this error call and the next one could use a second argument for the stack level - but ok to ignore